src/ansys/dpf/core/operator_specification.py

```diff
@@ -103,7 +104,7 @@ def type_names(self, val):
         self._type_names = val
 
     @staticmethod
-    def _get_copy(other, changed_types):
+    def _get_copy(other, changed_types) -> "PinSpecification":
```

Comment:
Why not the actual class?

Reply:
Because of imports problems. But they are fixed by importing annotations from __future__.